dbfit-java/derby/src/main/java/dbfit/environment/DerbyEnvironment.java

```diff
@@ -84,7 +85,80 @@ public Pattern getParameterPattern() {
 
     public Map<String, DbParameterAccessor> getAllProcedureParameters(
             String procName) throws SQLException {
-        throw new UnsupportedOperationException();
+        String[] qualifiers = procName.split("\\.");
```

Comment:
Ok. That's the existing name and it implements the method from the `DBEnvironment` interface.